rollup.config.js

```diff
@@ -21,60 +39,80 @@ let htmlPlugin = html ({
     },
 });
 
-let babelPlugin = babel({
-    exclude: 'node_modules/**/*',
-});
-
+// import LESS files as JS strings
 let lessPlugin = less({
     options: {
+        // TODO: centralize common config settings
         paths: [
-            'src/less', // TODO: centralize path in config
-        ]
-    }
+            'src/less',
+        ],
+    },
 });
 
-let eslintPlugin = eslint({
-    include: [
-        '**/*.js',
-    ],
+// import SASS/SCSS files as JS strings
+// see: https://sass-lang.com/documentation/js-api
+let sassPlugin = sass({
+    options: {
+        // TODO: centralize common config settings
+        includePaths: [
+            'src/scss',
+        ],
+        outputStyle: 'compact', // each style block on its own line (reduce generated white space)
+        precision: 4,
```

Comment:
This is NOT a magic number. This was determined through research to determine the accuracy of a rounded pixel calculation in extreme cases (e.g., 1 column of a 12-column grid at 360px vs 8K resolution).  A precision of 4 was shown to be accurate all the way up to 8K resolutions, whereas a precision of 3 would be accurate up to 4K.  A precision of 3 would be "good enough" but 4 is "perfect" and it _still_reduces the number of decimal places from 8 (thank you LESS) to 4, saving file size.